config/environments/development.rb

```diff
@@ -14,6 +14,19 @@
 
   config.action_controller.enable_fragment_cache_logging = true
 
+  # Enable/disable caching. By default caching is disabled.
+  # Run rails dev:cache to toggle caching.
+  if Rails.root.join("tmp/caching-dev.txt").exist?
```

Comment:
Shall we copy verbatim (except templating) from https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt#L20-L36?